cpp/include/ray/api/function_manager.h

```diff
@@ -49,7 +50,7 @@ inline static msgpack::sbuffer PackError(std::string error_msg) {
   msgpack::sbuffer sbuffer;
   msgpack::packer<msgpack::sbuffer> packer(sbuffer);
   packer.pack(msgpack::type::nil_t());
-  packer.pack(std::make_tuple((int)ray::rpc::ErrorType::TASK_EXECUTION_EXCEPTION,
+  packer.pack(std::make_tuple((int)ray::api::ErrorType::TASK_EXECUTION_EXCEPTION,
```

Comment:
Do we have to expose `ErrorType` here? It seems that it's cast to an `int` anyway.

Reply:
Do you mean use an integer value instead?

Reply:
Actually, I don't understand the code. It seems that all errors are converted to `TASK_EXECUTION_EXCEPTION`. Why?

Reply:
Because function_namnager has template classes, so the ErrorType can't be hidden in runtime